_posts/2020-10-01-hacktoberfest.markdown

```diff
@@ -3,6 +3,28 @@ title: Hacktoberfest and Void Linux
 layout: post
 ---
 
+*Update:* Unfortunately due to recent changes that significantly
+increase the effort required from Void, we can no longer promise that
+your changes will count towards the 4 PRs required for Hacktoberfest.
+
+Many maintainers use `git am` in a script to apply patches and batch
+them into build units that the buildbot can work on at a time.  This
+precludes the application of labels unless the maintainer separately
+opens a browser, logs into GitHub, and interacts with the PR there.
+While the `git am` method should technically will work, we've noticed
```

Comment:
Stike `will` from `should technically will work`

Reply:
good catch.